web/.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
-dist
\ No newline at end of file
+dist
+pkg/opni/generated
```

Comment:
We should version control generated code (unless it causes problems with node or something)

Reply:
Node shouldn't have an issue with it and I don't really have a strong opinion on it. In the past I've never checked in generated code and some of the arguments I've heard have been:
- Repos are mostly for tracking history and checking in the generated code makes the history less clear since the real change is in the generator.
- You're not losing anything by just checking in just the codegen since it should be idempotent and you can always generate again.
- Generated code isn't intended for reading

I'm fine to check it in but I'd like to heard why just since I've never done it.

Reply:
It is standard practice in the go ecosystem because go modules pull source code directly from git and without the generated code, the module won't be able to compile.

Reply:
For the typescript code it probably doesn't make much of a difference. Although I would say we should at least check in http_pb.ts to avoid needing to bootstrap the code generator every time.

Reply:
Added http_pb.ts.